File: src/main/java/com/sport/service/web/models/place/PlaceResponse.java
package com.sport.service.web.models.place;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PlaceResponse implements Serializable {

    private Long id;
    private String name;
    private String district;
    private String subDistrict;
    private String address;
    private String description;
    private String webSite;
    private String outdoor;
    private String placeType;
    private String coordinates;
    private String photoUrl;
}

File: src/main/kotlin/com/sport/service/web/models/place/PlaceResponse.kt
package com.sport.service.web.models.place

data class PlaceResponse(
    val name: String,
    val district: String,
    val subDistrict: String?,
    val address: String,
    val description: String?,
    val webSite: String?,
    val outdoor: String,
    val placeType: String,
    val photo: ByteArray,
    val coordinates: String?,
)